playbooks/adhoc/upgrades/upgrade.yml

```diff
@@ -50,6 +50,19 @@
         {{ openshift.common.admin_binary}} --config={{ openshift.common.config_base }}/master/admin.kubeconfig
         policy reconcile-cluster-roles --confirm
 
+- name: Update cluster policy bindings
+  hosts: oo_first_master
+  tasks:
+    - name: oadm policy reconcile-cluster-role-bindings --confirm
+      command: >
+        {{ openshift.common.admin_binary}} --config={{ openshift.common.config_base }}/master/admin.kubeconfig
+        policy reconcile-cluster-role-bindings
+        --exclude-groups=system:authenticated
+        --exclude-groups=system:unauthenticated
+        --exclude-users=system:anonymous
+        --additive-only=true --confirm
+  when: (_new_version.stdout > 1.0.6 and _new_version.stdout < 3.0) or _new_version.stdout > 3.0.2
```

Comment:
@sdodson Updated along with L43 and I switched it to operators so that it's hopefully less confusing for the advanced ones?